dataactvalidator/config/sqlrules/a19_cross_file.sql

```diff
@@ -1,11 +1,26 @@
 SELECT
-    approp.row_number,
-    approp.tas,
-    approp.obligations_incurred_total_cpe,
-    SUM(op.obligations_incurred_by_pr_cpe) as obligations_incurred_by_pr_cpe_sum
-FROM appropriation approp
-JOIN object_class_program_activity op
-ON approp.tas = op.tas AND approp.submission_id = op.submission_id
+	approp.row_number,
+   approp.allocation_transfer_agency,
+	approp.agency_identifier,
+	approp.beginning_period_of_availa,
+	approp.ending_period_of_availabil,
+	approp.availability_type_code,
+	approp.main_account_code,
+	approp.sub_account_code,
+	approp.obligations_incurred_total_cpe,
+	SUM(op.obligations_incurred_by_pr_cpe) as obligations_incurred_by_pr_cpe_sum
```

Comment:
Probably not an issue, but wanted to note it: the rules file says "negative sum"